src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/GetElevationAtAPoint/GetElevationAtAPoint.xaml.cs

```diff
@@ -0,0 +1,107 @@
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Get elevation at a point",
+        "Map",
+        "Retreive the elevation of a point on a surface",
+        "Tap anywhere on the surface to get the elevation at that point. Elevation is reported in meters since the scene view is in WGS84.")]
+    public partial class GetElevationAtAPoint
+    {
+        // URL of the elevation service - provides elevation component of the scene.
+        private readonly Uri _elevationUri = new Uri("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer");
+
+        // Starting point of the observer.
+        private readonly MapPoint _observerPoint = new MapPoint(83.9, 28.42, SpatialReferences.Wgs84);
+
+        // Create the graphics overlay.d
+        private GraphicsOverlay _overlay;
+
+        // Create symbols for the text and marker.
+        private SimpleMarkerSceneSymbol _elevationMarker;
+        private TextSymbol _elevationTextSymbol;
+        private readonly Graphic _elevationTextGraphic = new Graphic();
+
+        public GetElevationAtAPoint()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization.
+            Initialize();
+
+            // Handle taps on the scene view for getting elevation.
+            MySceneView.GeoViewTapped += SceneViewTappedAsync;
+        }
+
+        private void Initialize()
+        {
+            // Create the camera for the scene.
+            Camera camera = new Camera(_observerPoint, 30000.0, 10.0, 80.0, 0.0);
+
+            // Create scene.
+            Scene myScene = new Scene(Basemap.CreateImageryWithLabels())
+            {
+                // Set the initial viewpoint.
+                InitialViewpoint = new Viewpoint(_observerPoint, 1000000, camera)
+            };
+
+            // Create the marker for showing where the user taps.
+            _elevationMarker = SimpleMarkerSceneSymbol.CreateCylinder(Color.Red, 10, 750);
+
+            // Create the text for displaying the elevation value.
+            _elevationTextSymbol = new TextSymbol("", Color.Red, 20, Esri.ArcGISRuntime.Symbology.HorizontalAlignment.Center, Esri.ArcGISRuntime.Symbology.VerticalAlignment.Middle);
+            _elevationTextGraphic.Symbol = _elevationTextSymbol;
+
+            // Add base surface for elevation data.
+            myScene.BaseSurface.ElevationSources.Add(new ArcGISTiledElevationSource(_elevationUri));
+
+            // Create the graphics overlay.
+            _overlay = new GraphicsOverlay
+            {
+                SceneProperties = new LayerSceneProperties(SurfacePlacement.Absolute)
+            };
+            MySceneView.GraphicsOverlays.Add(_overlay);
+
+            // Add the scene to the view.
+            MySceneView.Scene = myScene;
+        }
+
+        private async void SceneViewTappedAsync(object sender, Esri.ArcGISRuntime.UI.Controls.GeoViewInputEventArgs e)
```

Comment:
Unhandled exceptions in async void methods can cause problems, so its best to wrap any calls in a try/catch.